src/libraries/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str.cs

```diff
@@ -205,26 +204,31 @@ public void HiddenFilesAreReturned()
         #region PlatformSpecific
 
         [Fact]
-        public void InvalidPath_Core()
+        public void InvalidPath()
         {
-            foreach (char invalid in Path.GetInvalidFileNameChars())
+            Assert.All(Path.GetInvalidFileNameChars(), invalidChar =>
             {
-                string badPath = string.Format($"{TestDirectory}{Path.DirectorySeparatorChar}te{invalid}st");
-                switch (invalid)
+                string badPath = string.Format($"{TestDirectory}{Path.DirectorySeparatorChar}te{invalidChar}st");
+
+                switch (invalidChar)
                 {
                     case '/':
                     case '\\':
-                    case ':':
                         Assert.Throws<DirectoryNotFoundException>(() => GetEntries(badPath));
                         break;
+                    case ':':
+                        // Framework does not support reading colon
```

Comment:
Let's spell out ".NET Core" and ".NET Framework" any time they occur.  We've been going through and fixing up existing mentions of them.
```suggestion
                        // .NET Framework does not support reading colon
```

Reply:
Sure, no problem. I'll do the same with all the cases.